README.md

```diff
@@ -225,11 +225,15 @@ PyPRT can be installed with `pip install pyprt` or `conda install -c esri pyprt`
   </tr>
 </table>
 
-## License
+## Licensing Information
 
-PyPRT is under the same license as the included [CityEngine SDK](https://github.com/Esri/esri-cityengine-sdk#licensing).
+PyPRT is free for personal, educational, and non-commercial use. Commercial use requires at least one commercial license of the latest CityEngine version installed in the organization. Redistribution or web service offerings are not allowed unless expressly permitted.
 
-An exception is the PyPRT source code (without CityEngine SDK, binaries, or object code), which is licensed under the Apache License, Version 2.0 (the “License”); you may not use this work except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+PyPRT is under the same license as the included [CityEngine SDK](https://github.com/Esri/esri-cityengine-sdk#licensing). An exception is the PyPRT source code (without CityEngine SDK, binaries, or object code), which is licensed under the Apache License, Version 2.0 (the “License”); you may not use this work except in compliance with the License. You may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0.
+
+All content in the "Examples" directory/section is licensed under the APACHE 2.0 license as well.
```

Comment:
Suggestion: Do we want to change the "in the "Examples" directory/section" with something like "in this repository" or something like that?

Reply:
Let's keep it generic everywhere, thanks.